src/extension/background-script/actions/setup/reset.ts

```diff
@@ -1,16 +1,11 @@
 import type { MessageReset } from "~/types";
 
+import db from "../../db";
 import state from "../../state";
 
 const reset = async (message: MessageReset) => {
-  await state.getState().password(null);
-  state.setState({
-    accounts: {},
-    account: null,
-    connector: null,
-    currentAccountId: null,
-  });
-  await state.getState().saveToStorage();
+  await state.getState().reset();
```

Comment:
I see that `reset()` will change back `accounts` and `currentAccountId` back to the default values, but I couldn't see if `account` and `connector` are being reset as well. Should they?

Reply:
@fczuardi thanks for the review! I think I have fixed the issue now.